app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+    def index
+        @foods = FoodFacade.get_foods(params[:search])
```

Comment:
you could give yourself access to totalHits and foods in here by doing this:
```
results = FoodFacade.get_foods(params[:search])
@total_hits = results[0]
@foods = results[1]
```
assuming that `get_foods` is giving us an array.